html/Front.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>index</title>
+    <link rel="stylesheet" href="../css/front.css">
+    
+    <meta charset="UTF-8"> 
+    <meta name="description" content="Front End for a level-up Events app">
+    <meta name="keywords"  lang"en-us"  content="events, event, app, level-up, andela">
+    <meta name="author" content="Mutesasira Moses">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1, user-scale-scalable=0">
+    <meta http-equiv ="X-UA-Compatible" content = "IE-edge, chrome = 1" >
+    <meta  name = "handheldfriendly" content = "true">
+  </head>
+  <body>
+        <div class="topnav">
+                <a class="active" href="#home">Home</a>
+                <a href="#news">Login</a>
+                <a href="#contact">Contact</a>
+                <a href="#about">About us</a>
+        </div>
+
+
+      <div  id = "left">
+          <div>
+         <img id= "image1" src="../images/event2.jpg">
+        </div>
+
+         <div>
+                <img id= "image2" src="../images/event.jpg">
+         </div>
+      </div>
+
+
+      <div  id = "form">
+      <form method = "POST">
+          <fieldset id = "field">
+              <legend>Login</legend>
+          <div>
+              <div class="input">
+                 <spnan> First name:</span>
+                    <input class="inputbutton1" type = "text" name = "first" placeholder= "first name"></input>
```

Comment:
input tags are self closing tags meaning u don't need `</input>` to close it either leave it as it is or end it with` />`